project/ProjectPlugin.scala

```diff
@@ -59,11 +68,53 @@ object ProjectPlugin extends AutoPlugin {
       )
     }
 
+    lazy val serverHttpDependencies = Seq(
+      libraryDependencies ++= Seq(
+        %%("circe-core", V.circe),
+        %%("circe-generic", V.circe),
+        %%("circe-parser", V.circe),
+        %%("log4s", V.log4s),
+        %("slf4j-simple", V.slf4jSimple),
+        %%("http4s-dsl", V.http4s),
+        %%("http4s-blaze-server", V.http4s),
+        %%("http4s-blaze-client", V.http4s),
+        %%("http4s-circe", V.http4s),
+        %("config"),
+        %%("jwt-core", V.jwtCore),
+        %%("scalatest", V.scalatest) % "test"
+      ),
+      addSbtPlugin("io.get-coursier" % "sbt-coursier" % "2.0.0-RC3-2")
```

Comment:
Is that the latest version of coursier? In any case, I'd define a variable for it too